### ISSUE TITLE
Feature Request for dark mode

### ISSUE THREAD

**Issue by kemalsaruhan** [huangyuzhang/Fizzy-Theme#56]
İs there any road map for dark mode? Thx a lot.

**Comment by huangyuzhang** [comment]
Yes, there will be a dark mode for Fizzy. However, I might change to other static site generators rather than Ghost to rebuild the theme.

**Comment by ugurcany** [comment]
Any update on this? Great theme and definitely waiting for dark mode as we have Ghost 4.0 now and it has a quite promising future.

**Comment by kemalsaruhan** [comment]
is static site Like a wordpress?

**Comment by denvergeeks** [comment]
https://www.google.com/search?q=static+site+generators

**Comment by mbuxmann** [comment]
> Yes, there will be a dark mode for Fizzy. However, I might change to other static site generators rather than Ghost to rebuild the theme.

So is the fizzy jam the new successor? Cause I wouldn't mind contributing to the dark theme. Something like this perhaps ? 
![darkmode fizzy](https://user-images.githubusercontent.com/21178318/116804830-49c7ca00-ab22-11eb-9304-2678752277fb.png)

**Comment by huangyuzhang** [comment]
> > Yes, there will be a dark mode for Fizzy. However, I might change to other static site generators rather than Ghost to rebuild the theme.
> 
> So is the fizzy jam the new successor? Cause I wouldn't mind contributing to the dark theme. Something like this perhaps ?
> ![darkmode fizzy](https://user-images.githubusercontent.com/21178318/116804830-49c7ca00-ab22-11eb-9304-2678752277fb.png)

Sorry for the late reply. 

Haha! Yes, Fizzy Jam will be the successor of this project. However, I currently could only spend limited time on both projects.
The screenshot seems really promising and contributions like this would be much appreciated!
